### ISSUE TITLE
Can't transforms when loading dataset classy_cifar10 or classy_cifar100

### ISSUE THREAD

**Issue by ddthuan** [facebookresearch/ClassyVision#591]
1. Config to load dataset Cifar  in file "ds_cifar.py":

from classy_vision.dataset import build_dataset, CIFARDataset
dataset_config = {
    "name": "classy_cifar10",
    "batchsize_per_replica": 10,
    "use_shuffle": True,
    "transforms": [
        {
            "name": "apply_transform_to_key",
            "transforms": [
                {"name": "ToTensor"},
                {"name": "Normalize", "mean": [0.485, 0.456, 0.406], "std": [0.229, 0.224, 0.225]}
            ],
            "key": "input"
        }
        ],
    "root":"data",    
    "train":True,
    "download":True,    
}

ds_cifar = build_dataset(dataset_config)
assert isinstance(ds_cifar, CIFARDataset) // pass
print(ds_cifar[0])

2. Run: python ds_cifar.py
**### ERROR:**
Files already downloaded and verified
Traceback (most recent call last):
  File "ds_cifar.py", line 24, in <module>
    print(ds_cifar[0])
  File "/home/photonic/anaconda3/envs/classify/lib/python3.7/site-packages/classy_vision/dataset/classy_dataset.py", line 122, in __getitem__
    return self.transform(sample)
  File "/home/photonic/anaconda3/envs/classify/lib/python3.7/site-packages/torchvision/transforms/transforms.py", line 61, in __call__
    img = t(img)
  File "/home/photonic/anaconda3/envs/classify/lib/python3.7/site-packages/classy_vision/dataset/transforms/util.py", line 76, in __call__
    sample
TypeError: '<' not supported between instances of 'str' and 'int'

## Environment
build from source
pytorch version: 1.6.0
classy_vision version: 0.5.0.dev

how to fix this bug?
Thank you.

**Comment by mannatsingh** [comment]
Hi @ddthuan ! The problem here is in the way you're specifying the transforms. Our [dataset tutorial](https://classyvision.ai/tutorials/classy_dataset) goes over configuring transforms. Since the underlying dataset is a torchvision dataset, [this](https://classyvision.ai/tutorials/classy_dataset#Transform-example-for-a-torchvision-dataset) is the relevant section. I would recommend going over the whole tutorial though.

Basically, `apply_transform_to_key` alone isn't sufficient here - you need to use `generic_image_transform` or compose `apply_transform_to_key` and `tuple_to_map `, as per the tutorial.

**Comment by ddthuan** [comment]
Thank @mannatsingh. It's okay. I will read the tutorial more carefully. 
Many thank you.
